tests/test_config.py

```diff
@@ -80,3 +83,30 @@ def test_grocker_yml(self):
             config = parse_config([])
             self.assertIn('not_used_key', config)  # .grocker.yml is read
             self.assertIn('entrypoint_name', config)  # grocker internal config is read
+
+
+class DefaultImageNameTC(unittest.TestCase):
+    def test_default_image_name(self):
+        releases = ('grocker-test-project==2.0.0', 'grocker-test-project[pep8]==2.0.0')
+        image_names = (None, 'demo-app')
+        docker_prefixes = (None, 'registry.local')
+        product = itertools.product(releases, image_names, docker_prefixes)
+        expected_names = [
+            'grocker-test-project:2.0.0-',
```

Comment:
nit: At first glance, it seems strange, we expect `grocker-test-project:2.0.0-{grocker_version}` and not `grocker-test-project:2.0.0-`. Maybe `grocker-test-project:2.0.0-%s` and replacing the `+` operator by `%` at line 112, will be more readable, right ?